tools/cockpit.spec

```diff
@@ -366,6 +366,7 @@ Requires: kexec-tools
 Recommends: polkit
 Recommends: NetworkManager-team
 Recommends: setroubleshoot-server >= 3.3.3
+Recommends: (reportd if abrt)
```

Comment:
That will add the recommends for RHEL only. Please move it outside the `%if` so that it also applies to Fedora.

Reply:
Oops.